<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ZSpeexDialog</class>
 <widget class="QDialog" name="ZSpeexDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>491</width>
    <height>199</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Speex plugin parameters</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <layout class="QGridLayout" name="gridLayout">
     <item row="0" column="0">
      <widget class="QCheckBox" name="checkDenoise">
       <property name="text">
        <string>Denoise</string>
       </property>
       <property name="tristate">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item row="0" column="1">
      <widget class="QCheckBox" name="checkAGC">
       <property name="text">
        <string>Automatic gain control (AGC)</string>
       </property>
       <property name="tristate">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item row="1" column="0">
      <widget class="QCheckBox" name="checkDereverb">
       <property name="text">
        <string>Dereverb</string>
       </property>
       <property name="tristate">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item row="1" column="1">
      <widget class="QCheckBox" name="checkEcho">
       <property name="text">
        <string>Echo canceller</string>
       </property>
       <property name="tristate">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item row="2" column="0">
      <widget class="QCheckBox" name="checkFrames">
       <property name="text">
        <string>Frames</string>
       </property>
      </widget>
     </item>
     <item row="2" column="1">
      <widget class="QSpinBox" name="spinFrames">
       <property name="toolTip">
        <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;This controls the frames of the intermediate buffer.  This corresponds to the latency of the filter.&lt;/p&gt;&lt;p&gt;Number of samples to process at one time (should correspond to 10-20 ms).&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
       </property>
       <property name="minimum">
        <number>1</number>
       </property>
       <property name="maximum">
        <number>65536</number>
       </property>
       <property name="singleStep">
        <number>16</number>
       </property>
       <property name="value">
        <number>64</number>
       </property>
      </widget>
     </item>
     <item row="3" column="0">
      <widget class="QCheckBox" name="checkAGCLevel">
       <property name="text">
        <string>AGC level</string>
       </property>
      </widget>
     </item>
     <item row="3" column="1">
      <widget class="QSpinBox" name="spinAGCLevel">
       <property name="minimum">
        <number>1</number>
       </property>
       <property name="maximum">
        <number>65536</number>
       </property>
       <property name="value">
        <number>8000</number>
       </property>
      </widget>
     </item>
     <item row="4" column="0">
      <widget class="QCheckBox" name="checkFilterLength">
       <property name="text">
        <string>Echo canceller filter length</string>
       </property>
      </widget>
     </item>
     <item row="4" column="1">
      <widget class="QSpinBox" name="spinFilterLength">
       <property name="toolTip">
        <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;Number of samples of echo to cancel.&lt;/p&gt;&lt;p&gt;Should generally correspond to 100-500 ms.&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
       </property>
       <property name="minimum">
        <number>1</number>
       </property>
       <property name="maximum">
        <number>65536</number>
       </property>
       <property name="singleStep">
        <number>16</number>
       </property>
       <property name="value">
        <number>256</number>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QDialogButtonBox" name="buttonBox">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="standardButtons">
      <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <tabstops>
  <tabstop>checkDenoise</tabstop>
  <tabstop>checkAGC</tabstop>
  <tabstop>checkDereverb</tabstop>
  <tabstop>checkEcho</tabstop>
  <tabstop>checkFrames</tabstop>
  <tabstop>spinFrames</tabstop>
  <tabstop>checkAGCLevel</tabstop>
  <tabstop>spinAGCLevel</tabstop>
  <tabstop>checkFilterLength</tabstop>
  <tabstop>spinFilterLength</tabstop>
 </tabstops>
 <resources/>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>ZSpeexDialog</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>227</x>
     <y>179</y>
    </hint>
    <hint type="destinationlabel">
     <x>157</x>
     <y>198</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>ZSpeexDialog</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>295</x>
     <y>185</y>
    </hint>
    <hint type="destinationlabel">
     <x>286</x>
     <y>198</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
